src/dirigera/devices/air_purifier.py

```diff
@@ -0,0 +1,125 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any, Optional, Dict, List, Union
+
+from .device import Device
+
+from ..hub.abstract_smart_home_hub import AbstractSmartHomeHub
+
+
+class FanModeEnum(Enum):
+    LOW = "low"
+    MEDIUM = "medium"
+    HIGH = "high"
+    AUTO = "auto"
+
+
+@dataclass
+class AirPurifier(Device):
+    dirigera_client: AbstractSmartHomeHub
+    is_reachable: bool
+    fan_mode: str
+    fan_mode_sequence: str
+    child_lock: Optional[bool]
+    status_light: Optional[bool]
+    motor_runtime: int
+    motor_state: int
+    filter_alarm_status: bool

```

Comment:
I renamed the attribute, since the new name is easier to understand. Explanation for the original name: I based all names on how the values are called in the API.